src/screens/internal/CompareSnapshots/AnomalyModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { AnomalyModalWrapper, AnomalyButton } from './AnomalyCompare.style';
+import { Modal, Typography } from '@material-ui/core';
+import { ProjectionsPair } from 'common/models/ProjectionsPair';
+import { Metric } from 'common/metricEnum';
+import { getAnomaliesForMetric } from './utils';
+
+export default function AnomalyModal({
+  pair,
+  metric,
+}: {
+  pair: ProjectionsPair;
+  metric: Metric;
+}) {
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+
+  const leftAnomalies = getAnomaliesForMetric(pair.left, metric);
+  const rightAnomalies = getAnomaliesForMetric(pair.right, metric);
+  const newAnomalies = leftAnomalies?.length !== rightAnomalies?.length;
```

Comment:
Just as a flag, I think it's possible to remove anomalies between snapshots too, so "new" isn't necessarily accurate.  It's also probably technically possible that we gain and lose the same number of anomalies.  We might want to use isEqual() from lodash to do a deep comparison or something.

Reply:
Yeah good call, updated to use lodash `isEqual()` in 142e41ea069eac537f5ed8968120603f4f7ac33f